src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawing {
```

Comment:
```suggestion
public class Circle extends Figure {
```
зроби щоб в тебе клас Figure реалізовував Drawing і не потрібно буде це вручну в кожному класі робити